packages/serverless-logic-sandbox/static/samples/swf-report/swf-report.dash.yml

```
@@ -314,40 +356,20 @@ pages:
                           - source: duration
                             function: AVERAGE
                             column: Total
-      - columns:
-          - span: "12"
-            components:
-              - properties:
-                  width: "150px"
-                  margin-top: "20px"
-                  float: "right"
-                settings:
-                  type: SELECTOR
-                  general:
-                    visible: "true"
-                    text: "Filter"
-                  filter:
-                    enabled: "true"
-                    notification: "true"
-                  dataSetLookup:
-                    uuid: processInstances
-                    group:
-                      - columnGroup:
-                          source: processId
-                        groupFunctions:
-                          - source: processId
-                            column: Workflow
-          - layoutComponents:
+          - components:
               - html: "<hr />Workflows List"
                 properties:
                   font-size: x-large
                   margin-bottom: 20px
       - columns:
           - components:
-              - settings:
-                  type: TABLE
+              - settings:                
```

```suggestion
              - settings:
```
Can you please remove these extra white spaces? 😛